src/pretix/control/templates/pretixcontrol/organizers/customer.html

```diff
@@ -22,54 +22,70 @@ <h3 class="panel-title">
                     </h3>
                 </div>
                 <div class="panel-body">
-                    <form action="" method="post">
+                    <form action="" method="post" class="row">
```

Comment:
I know we talked about adding the row-class to the form, but later we talked about leaving the form alone and using one `<dl>` element to contain all the info and use two `<div>` to create the two columns – `<div>` is allowed inside `<dl>` to group pairs of `<dt><dd>`. So:
```suggestion
                    <form action="" method="post">
```